var/spack/repos/builtin/packages/libceed/package.py

```diff
@@ -129,18 +126,19 @@ def common_make_opts(self):
 
         return makeopts
 
-    def build(self, spec, prefix):
-        makeopts = self.common_make_opts
-        make('info', *makeopts)
-        make(*makeopts)
+    def edit(self, spec, prefix):
+        make('info', *self.common_make_opts)
 
-        if self.run_tests:
-            make('prove', *makeopts, parallel=False)
+    @property
+    def build_targets(self):
+        return self.common_make_opts
 
-    def install(self, spec, prefix):
-        installopts = ['prefix=%s' % prefix]
-        installopts += self.common_make_opts
-        make('install', *installopts, parallel=False)
+    @property
+    def install_targets(self):
+        return ['prefix={0}'.format(self.prefix)] + self.common_make_opts
+
+    def check(self):
+        make('prove', *self.common_make_opts, parallel=False)
```

Comment:
I do we assume people will have `prove` (from perl) or should we use `make test` here?